xml/dhcp_vendor_class.xml

```diff
@@ -171,6 +171,15 @@
     <param pos="0" name="os.cpe23" value="cpe:/o:google:android:{os.version}"/>
   </fingerprint>
 
+  <fingerprint pattern="HUAWEI:android:" certainty="0.9">
```

Comment:
Can we add a start of string anchor here? What is the "ABC" value in the example and can it be placed into the regex so that we can also have an end of string anchor?

Reply:
Actual DHCP strings include what appears to be a random string, sometimes made up of 3 characters, sometimes of more:

- HUAWEI:android:ANE
- HUAWEI:android:FIG-L11
- HUAWEI:android:FIG-L21
- HUAWEI:android:LYA

Reply:
Do you have any idea what the string represents? Could it be made a part of the pattern and allow us to add an end of string anchor?